packages/core/src/BasicResources.ts

```diff
@@ -80,7 +78,7 @@ export class BasicResources {
     mesh._addReferCount(1);
     mesh.setVertexElements([new VertexElement("POSITION_UV", 0, VertexElementFormat.Vector4, 0)]);
     mesh.setVertexBufferBinding(new Buffer(engine, BufferBindFlag.VertexBuffer, vertices, BufferUsage.Static), 16);
-    mesh.addSubMesh(0, 4, MeshTopology.TriangleStrip);
+    mesh.addSubMesh(0, 3, MeshTopology.Triangles);
```

Comment:
Add this comments: https://michaldrobot.com/2014/04/01/gcn-execution-patterns-in-full-screen-passes/

Reply:
done